### ISSUE TITLE
Consider adding an aria-label to the "search" landmark

### ISSUE THREAD

**Issue by jvine** [projectblacklight/blacklight#2147]
`role="search"` defines the search landmark in Blacklight. 👍 

However some SRs don't list landmarks that don't have an aria-label. 
- VoiceOver does not list the search landmark.
- NVDA does list it. 

Consider adding `aria-label="search"` to `<form role="search" ...>` for consistency of support.

**Comment by cjcolvar** [comment]
@barmintor Can this be closed by #2603?
